ui/sites/compendium-site/node_modules/.vite/deps/_metadata.json

```diff
@@ -0,0 +1,62 @@
+{
+  "hash": "eeac3eee",
+  "browserHash": "9c81bcee",
+  "optimized": {
```

Comment:
Not sure how vite works - do we want to commit all of these files including node_modules and ts-maps?

Reply:
removed